lib/sandbox.js

```diff
@@ -113,14 +113,28 @@ function init ( ) {
     }
   };
 
+  sbx.lastEntry = function lastEntry (entries) {
```

Comment:
Really like this type of API calls on the sandbox, which guarantee bug free access to the data